### PR TITLE
Add ingress pod to logs output

### PR DESCRIPTION
**Pull request by spowelljr**
- Add ingress pod to logs output
- Refactor logic to make it easy to add new addons to logs in the future
- Move existing addon pods to new logic to save checking for pod that may not exist

### CODE REVIEW

File: pkg/minikube/logs/logs.go [hunk 1]
```diff
@@ -291,3 +288,19 @@ func logCommands(r cruntime.Manager, bs bootstrapper.Bootstrapper, cfg config.Cl
 
 	return cmds
 }
+
+func enabledAddonPods(cfg config.ClusterConfig) []string {
```
Comment (medyagh) [inline pkg/minikube/logs/logs.go:292]: add comment, the name is not clear what does it do.

Comment (spowelljr) [reply pkg/minikube/logs/logs.go:294]: Added a comment

**Review by YasinAshorian — APPROVED** [review]

**Review by medyagh — APPROVED** [review]

**Comment by medyagh** [comment]
merge after tests are passing

**Pull request merged**